services/movie_session.py

```diff
@@ -42,3 +42,12 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id : int) -> list:
```

Comment:
Consider using `List[Dict[str, int]]` as a return type hint for the `get_taken_seats` function to indicate it returns a list of dictionaries with `row` and `seat` keys.